inc/class-catalog.php

```diff
@@ -754,7 +754,12 @@ function saveProfile( array $item ) {
 			} elseif ( '%f' === $this->profileMetaKeys[ $key ] ) {
 				$val = (float) $val;
 			} else {
-				$val = (string) $val;
+				$val = HtmLawed::filter(
```

Comment:
This code allows to input html tag and removes the "style" attribute if present, I think this is the intended behavoir.

Reply:
Yeah. It's intended. I went with this recommendation:

> Admins, however, may still want to completely deny the style attribute, e.g., with code like
> `$processed = htmLawed($text, array('safe'=>1, 'deny_attribute'=>'style'));`
> Permitting the style attribute brings in risks of click-jacking, etc. CSS property values can render a page non-functional or be used to deface it. 

Source:
http://www.bioinformatics.org/phplabware/internal_utilities/htmLawed/htmLawed_README.htm#s3.6